src/js/hopscotch.js

```diff
@@ -1945,6 +1945,8 @@
     this.showStep = function(stepNum) {
       var step = currTour.steps[stepNum];
       if(!utils.getStepTarget(step)) {
+        currStepNum = stepNum;
```

Comment:
Let's remove this. IMO, if the step's target doesn't exist, we should leave the tour where it is. This also is probably the source of the build failure.